lib/cretonne/src/ir/function.rs

```diff
@@ -141,6 +141,8 @@ impl Function {
     }
 
     /// Return an object that can display this function with correct ISA-specific annotations.
+    // Needless lifetimes lint false positive
+    #[cfg_attr(feature = "cargo-clippy", allow(needless_lifetimes))]
```

Comment:
It feels like needless_lifetimes has enough false positives that it'd be better to disable this lint globally than clutter up the code disabling it locally every time.

Reply:
agreed